nncf/experimental/common/tensor_statistics/collectors.py

```diff
@@ -311,15 +307,17 @@ def get_statistics(self) -> Union[TensorStatistic, Dict[str, Any]]:
 
         :returns: Aggregated values.
         """
-
+        if self.statistics:
+            return self.statistics
```

Comment:
I thought about such solution, glad it is implemented!
I have one concern:  tensor collector can still accumulate statistics after the `get_statistics` call, which could alter the return value of `get_statistics`. So, we can do one of two things:
   * Keep cached statistics as a dict `current_num_samples -> statistics`
   * Disable aggregator after the `get_statistics` call
As there is no use cases when tensor collector collects half of statistics, dumps it and collect more, I believe second approach should be sufficient: just call `self.disable()` in `get_statistics` method

Reply:
added disable()

Reply:
added a test